### PR TITLE
Migration to use zilliqajs utils

### LINKED ISSUE #26: Use util methods directly from zilliqa-js
zilliqa-js-0.1.2 has new crypto stuffs and smaller build size which has to be linked up in kaya.

### PR DESCRIPTION
**Pull request by edisonljh**
## Description
- Kaya RPC now uses util methods from `zilliqa-js` directly. Previously. Kaya RPC uses a localized version of the util files as the old javascript library is huge. Resolves #26 
- Removed dependency on `bcrypto`: `bcrypto` is a large library and Kaya RPC only uses one function within `sha256`. This PR switches to `hash.js` sha256 implementation.
- Changed log-level naming from `debug_txn` to `LOG_[script name]` for better readability. This change follows the core zilliqa coding convention.
- Added option to run debug mode through npm: `npm run debug`

## Review Suggestion
Since there's no local test, we have to do reviews through the manual way.

1. Start the server `npm run debug` Copy one of the keys.
2. On another terminal, go to `test/scripts` and run the script using the command `node DeployContract.js --key [private_key]`.

### Implementation
- [x] **ready for review**

### CODE REVIEW

**Comment by edisonljh** [comment]
Also included bug fixes where a transaction fails if the recipient address is not created in through kaya.

**Comment by iantanwx** [comment]
I just have one question - how come there is a swap file in the repo?

File: logic.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
 const sha256 = require("bcrypto").sha256;
```
Comment (iantanwx) [inline logic.js]: @edisonljh I think we should use `hashjs` for consistency.

Comment (edisonljh) [reply logic.js]: Working on it now, will push to the next PR.

**Pull request merged**